app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -353,6 +353,8 @@ def add_secret_to_pod_def(pod_def, inspector_admin_secret_name)
   end
 
   def inspector_image
-    'docker.io/openshift/image-inspector:v2.0.z'
+    registry = Settings.ems.ems_kubernetes.image_inspector_registry || INSPECTOR_REGISTRY_FALLBACK
+    repo = Settings.ems.ems_kubernetes.image_inspector_repository || INSPECTOR_REPOSITORY_FALLBACK
```

Comment:
Current code handles the possibility of leaves missing(e.g :image_inspector_registry) and it would make sense  to handle upper level elements as well (e.g ems_kubernetes) I recently came across following code in miq_schedule_worker, maybe you can use something similar?

https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_schedule_worker/runner.rb#L66